README.md

```
@@ -239,21 +266,21 @@ Yes! If you encounter additional false positives:
-- **always**: Automatically merge existing suppressions
-- **never**: Always overwrite the `.clangd` file
-
-When merging, your custom suppressions are combined with the default MQL suppressions without duplicates. Set this in your VS Code settings (`.vscode/settings.json` or global settings):
-
-```json
-{
-    "mql_tools.Clangd.PreserveSuppressions": "always"
-}
-```
-
+**Preserving Custom Diagnostic Suppressions**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)